lib/constants.js

```diff
@@ -37,7 +37,8 @@ module.exports = Object.freeze({
   },
   HTTP_STATUS_CODE: {
     CONFLICT: 409,
-    UNPROCESSABLE_ENTITY: 422
+    UNPROCESSABLE_ENTITY: 422 ,
+    BAD_REQUEST : 400
```

Comment:
A minor edit, can we follow the same pattern for line number 40, 41 as in line number 39.